save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/cosv/RawCosvFile.kt

```diff
@@ -0,0 +1,65 @@
+package com.saveourtool.save.entities.cosv
+
+import com.saveourtool.save.entities.Organization
+import com.saveourtool.save.entities.User
+import com.saveourtool.save.spring.entity.BaseEntityWithDtoWithId
+import com.saveourtool.save.spring.entity.IBaseEntityWithDate
+import java.time.LocalDateTime
+import javax.persistence.Entity
+import javax.persistence.Enumerated
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+
+/**
+ * Entity for table `raw_cosv_file`
+ *
+ * @property fileName
+ * @property user
+ * @property organization
+ * @property status
+ * @property createDate
+ * @property updateDate
+ * @property statusMessage
+ */
+@Entity
+@Suppress("LongParameterList")
+class RawCosvFile(
```

Comment:
Better name this class with `Entity` postfix, no?

Reply:
it's a file, not entity. user uploads files and then they are processed to entities